### PR TITLE
Post-Review Cleanup

### PR DESCRIPTION
**Pull request by jefftune**
@tunezaq
- Code review cleanup from https://github.com/zaquestion/requests-mv-integrations/pull/1#pullrequestreview-9589858
- Code formatted by yapf: https://github.com/google/yapf, note new file **.style.yapf**
- Still wish to talk about exception handling.

### CODE REVIEW

**Comment by jefftune** [comment]
@tunezaq 
- Applied **yapf** to code and examples, and added **.style.yapf**
- Modified **.pep8**
- included changes from **requests-mv-integrations**
- using **session** generated from **TuneRequest**

File: requests_mv_integrations/request_mv_integration.py [hunk 1]
```diff
@@ -15,60 +15,34 @@
...
-    TuneRequestClientError,
-    TuneRequestServiceError,
-    TuneRequestModuleError,
-
-    get_exception_message,
-    print_traceback,
-
-    TuneIntegrationExitCode
+    TuneRequestError, TuneRequestClientError, TuneRequestServiceError, TuneRequestModuleError, get_exception_message,
+    print_traceback, TuneIntegrationExitCode, ModuleArgumentError
```
Comment (zaquestion) [inline requests_mv_integrations/request_mv_integration.py]: adding the trailing comma here will make these imports use line splitting.

File: requests_mv_integrations/request_mv_integration_upload.py [hunk 2]
```diff
@@ -7,45 +7,22 @@
...
-    RequestMvIntegration
+    base_class_name, python_check_version, REQUEST_RETRY_EXCPS, REQUEST_RETRY_HTTP_STATUS_CODES
 )
+from requests_mv_integrations import (__python_required_version__)
+from .request_mv_integration import (RequestMvIntegration)
 
 python_check_version(__python_required_version__)
 
 
 class RequestMvIntegrationUpload(RequestMvIntegration):
```
Comment (zaquestion) [inline requests_mv_integrations/request_mv_integration_upload.py]: Same as download, should sever inheritance and make a ref on the class. Consuming into RequestMvIntegration is fine as well.

**Comment by zaquestion** [comment]
Few comments, otherwise LGTM

**Comment by zaquestion** [comment]
also again on releases

Please no dist directory, these are non-diffable refs that only bloat the git tree

Instead create a release and upload the egg:
https://help.github.com/articles/creating-releases/

To create the release (these will work with pypi)
https://github.com/TuneLab/requests-mv-integrations/releases

**Pull request merged**